src/remote_adapter/remote_adapter_client/qt_impl/sdl_remote_adapter_qt_client.cc

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+
+#include "qt_impl/sdl_remote_adapter_qt_client.h"
+#include "qt_impl/sdl_remote_adapter_receive_thread.h"
+
+namespace lua_lib {
+
+SDLRemoteTestAdapterQtClient::SDLRemoteTestAdapterQtClient(
+            const std::string& host,
+            uint32_t port,
+            MqParams& in_params,
+            MqParams& out_params,
+            QObject* parent)
+        : QObject(parent),
+          host_(host),
+          port_(port),
+          in_mq_params_(in_params),
+          out_mq_params_(out_params)  { }
+
+SDLRemoteTestAdapterQtClient::~SDLRemoteTestAdapterQtClient() {
+    if (nullptr != listener_ptr_) {
+        delete(listener_ptr_);
+    }
+
+    if (nullptr != remote_adapter_client_ptr_) {
+        remote_adapter_client_ptr_->clear();
```

Comment:
the same like above

Reply:
@KVGrygoriev Applied (see 1c6f54a5a5c135f34813d35ec8dc5521604899f0)